From c647fc7dafe7f1d1f6487b5f0b0dcfdb2744108f Mon Sep 17 00:00:00 2001
From: Xun Sun <xun.sun@intel.com>
Date: Wed, 13 Nov 2013 18:00:45 +0800
Subject: Enable SSE optimized IDCT in jpeg decoding for x86 platform.

BZ: 152325

Category: AOSP Improvement
Domain: AOSP.Optimization-Browser
Origin: Internal
Upstream-Candidate: Yes

Change-Id: I78e45fa1d72babea8e199fd9a945777ab7eaa13e
Orig-MCG-Change-Id: I90e979a72c1b10cde40526f97bdab3f128a236ac
Signed-off-by: Xun Sun <xun.sun@intel.com>
---
 Android.mk |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Android.mk b/Android.mk
index 6e78c74..b5905d1 100644
--- a/Android.mk
+++ b/Android.mk
@@ -35,7 +35,7 @@ LOCAL_CFLAGS += -O3 -fstrict-aliasing -fprefetch-loop-arrays
 # enable tile based decode
 LOCAL_CFLAGS += -DANDROID_TILE_BASED_DECODE
 
-ifeq ($(TARGET_ARCH_VARIANT),x86-atom)
+ifeq ($(TARGET_ARCH),x86)
   LOCAL_CFLAGS += -DANDROID_INTELSSE2_IDCT
   LOCAL_SRC_FILES += jidctintelsse.c
 endif
-- 
1.7.4.1

